core/iwasm/libraries/wasi-nn/src/wasi_nn_private.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2019 Intel Corporation.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#ifndef WASI_NN_PRIVATE_H
+#define WASI_NN_PRIVATE_H
+
+#include "wasi_nn_types.h"
+
+typedef struct {
+    bool is_initialized;
+    graph_encoding current_encoding;
```

Comment:
The allocated resources are not stored in the structure, we will implement that in the future, so as to support running multiple wasm apps at the same, right?

Reply:
exactly